js/templates/modals/settings/general.html

```diff
@@ -1,8 +1,8 @@
 <div class="bar padMd clrBr borderBottom">
   <h2 class="h3 txUnb"><%= ob.polyT('settings.generalTab') %></h2>
 </div>
-<div class="padMd tabFormWrapper clrS">
-  <form class="box border clrP clrBr padMdKids padStack settingsGeneralForm">
+<div class="tabFormWrapper clrP">
```

Comment:
tabFormWrapper is being used in _settings.scss. We shouldn't remove it unless that file were adjusted to.